Description: Remove debian-requirements.txt from MANIFEST exclude, no longer used
---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- conn-check-1.0.13.orig/MANIFEST.in
+++ conn-check-1.0.13/MANIFEST.in
@@ -1,7 +1,6 @@
 include LICENSE
 include CHANGELOG
 include *.txt
-exclude debian-requirements.txt
 exclude deb-dependencies.txt
 recursive-include conn_check *.txt
 recursive-include conn_check *.xml
